lc-hax/Scripts/Commands/SuitCommand .cs

```diff
@@ -0,0 +1,28 @@
+using Hax;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+[Command("/suit")]
+public class SuitCommand : ICommand {
+
+    public Dictionary<string, Unlockable> SuitUnlockables
+    => Enum.GetValues(typeof(Unlockable))
+            .Cast<Unlockable>()
+            .Where(u => u.ToString().EndsWith("_SUIT"))
+            .ToDictionary(suit => suit.ToString().Replace("_SUIT", string.Empty).ToLower(), suit => suit);
+
+    public void Execute(StringArray args) {
+        if (Helper.StartOfRound is not StartOfRound startOfRound) return;
+        if (args.Length == 0 || string.IsNullOrWhiteSpace(args[0])) {
```

Comment:
I think it's fine to not check for empty string. It'll just default to the first suit.

```suggestion
        if (args.Length == 0) {
```

Reply:
Yeah, it defaults to orange even if you put a extra space, someone can do this and this way it dumps the accessible suits.

Reply:
What's wrong with that? It's an unintended and quick way to go back to the orange suit.